cengage/react-magma
# feat(Tabs, NavTabs): Add support for overwriting `TextTransform`.

Issue: #1107

## What I did

Add support for overwriting `TextTransform` for `Tabs` and `NavTabs`.

## Screenshots:


## Checklist 

## How to test

Open Tabs/NavTabs in docs/storybook -> Set `TextTransform` property -> Check styles.
